src/ImageSharp/ImageFrame{TPixel}.cs

```diff
@@ -52,15 +51,47 @@ internal ImageFrame(MemoryManager memoryManager, int width, int height, ImageFra
             this.MetaData = metaData;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ImageFrame{TPixel}" /> class.
+        /// </summary>
+        /// <param name="configuration">The <see cref="Configuration"/> to use for buffer allocation and parallel options to clear the buffer with.</param>
+        /// <param name="width">The width of the image in pixels.</param>
+        /// <param name="height">The height of the image in pixels.</param>
+        /// <param name="backgroundColor">The color to clear the image with.</param>
+        internal ImageFrame(Configuration configuration, int width, int height, TPixel backgroundColor)
```

Comment:
This is good! I actually wanted to introduce the configuration as a parameter and drop the `MemoryManager` overload so we're halfway there.